### PR TITLE
Added HTML and some styling for compare screen.

### PR DESCRIPTION
**Pull request by ctaylo37**
Basic HTML framework in place for compare feature.  Needs custom header and functionality wired up.

### CODE REVIEW

**Comment by ctaylo37** [comment]
Here's the mock-up to go off of:
screen shot 2015-07-13 at 9 44 33 am

**Comment by rajadain** [comment]
Cherry picked #572 into this branch to fix JavaScript issues

**Comment by lewfish** [comment]
I would prefer it if the width of the scenarios was responsive, and all the scenarios would expand so as to fill up the window. There would have to be a min-width though on each scenario, so that you would have to scroll horizontally if there were too many scenarios (or too small a screen). I'm not sure how difficult this would be to implement. +1 otherwise.

**Comment by mmcfarland** [comment]
Good points @lewfish.  @ctaylo37 is that going to be easier to do in this prototype before we make it all dynamic?

**Comment by ctaylo37** [comment]
Most Likely

On Tue, Jul 14, 2015 at 1:43 PM, Matthew McFarland <notifications@github.com
> wrote:

> Good points @lewfish <https://github.com/lewfish>. @ctaylo37
> <https://github.com/ctaylo37> is that going to be easier to do in this
> prototype before we make it all dynamic?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WikiWatershed/model-my-watershed/pull/573#issuecomment-121319997>
> .
>



-- 
Christopher Taylor, UI/UX Designer

Azavea |  340 N 12th St, Ste 402, Philadelphia, PA
ctaylor@azavea.com  | T 215.925.2600  | F 215.925.2663
Web azavea.com <http://www.azavea.com/>  |  Blog azavea.com/blogs
<http://www.azavea.com/Blogs>  | Twitter @azavea <http://twitter.com/azavea>

**Comment by lewfish** [comment]
I'm creating views based on the HTML here. So, if you change anything, please make sure to put it in a separate commit so it's easy to update what I've done.

**Comment by ctaylo37** [comment]
I'm working on changes now.  I will put them in a new commit.

File: src/mmw/apps/home/templates/home/compare.html [hunk 1]
```diff
@@ -1,80 +1,241 @@
...
+                        </div>
+                    </td>
+                    <td> <!-- Scenario 2 -->
+                        <div class="compare-column"> <!-- Column containing scenario info -->
+
+                            <div class="map-container"> <!-- Map of scenario 2 -->
+                                <div class="scenario-title">
+                                    <h3 class="light">Scenario 2</h3>
+                                </div>
+                                <div id="map"></div>
```
Comment (lliss) [inline src/mmw/apps/home/templates/home/compare.html]: Same id is used on line 17.

File: src/mmw/apps/home/templates/home/compare.html [hunk 2]
```diff
@@ -1,80 +1,241 @@
...
+                        </div>
+                    </td>
+                    <td> <!-- Scenario 3 -->
+                        <div class="compare-column"> <!-- Column containing scenario info -->
+
+                            <div class="map-container"> <!-- Map of scenario 3 -->
+                                <div class="scenario-title">
+                                    <h3 class="light">Scenario 3</h3>
+                                </div>
+                                <div id="map"></div>
```
Comment (lliss) [inline src/mmw/apps/home/templates/home/compare.html]: id map already used.

File: src/mmw/apps/home/templates/home/compare.html [hunk 3]
```diff
@@ -1,80 +1,241 @@
...
+                        </div>
+                    </td>
+                    <td> <!-- Scenario 4 -->
+                        <div class="compare-column"> <!-- Column containing scenario info -->
+
+                            <div class="map-container"> <!-- Map of scenario 4 -->
+                                <div class="scenario-title">
+                                    <h3 class="light">Scenario 4</h3>
+                                </div>
+                                <div id="map"></div>
```
Comment (lliss) [inline src/mmw/apps/home/templates/home/compare.html]: id map already used.

File: src/mmw/apps/home/templates/home/compare.html [hunk 4]
```diff
@@ -1,80 +1,241 @@
...
+                        </div>
+                    </td>
+                    <td> <!-- Scenario 5 -->
+                        <div class="compare-column"> <!-- Column containing scenario info -->
+
+                            <div class="map-container"> <!-- Map of scenario 5 -->
+                                <div class="scenario-title">
+                                    <h3 class="light">Scenario 5</h3>
+                                </div>
+                                <div id="map"></div>
```
Comment (lliss) [inline src/mmw/apps/home/templates/home/compare.html]: again...

**Comment by ctaylo37** [comment]
@lliss the scenarios were duplicated with the same markup since this is just a prototype.  The map id's will need to come from the corresponding scenarios in model view.

**Comment by ctaylo37** [comment]
The header will need to be designed in a future PR.

**Comment by lewfish** [comment]
Why'd you decide to go with the navigation buttons instead of a scroll bar? I'm guessing so that the user doesn't need to do the work of scrolling just the right amount.

**Comment by lewfish** [comment]
I incorporated these commits into my PR #591, so I think it's safe to close this PR.

**Comment by rajadain** [comment]
@lewfish Were the slide transitions added in #596? If so, it should be safe to close this PR.

**Comment by lewfish** [comment]
Yes.\

**Pull request closed without merging**